wscript

```diff
@@ -405,6 +408,9 @@ def configure(conf):
 		else:
 			for i in a:
 				conf.check_cc(lib = i)
+	elif conf.env.DEST_OS == 'openbsd':
+		#libdl is a part of libc on this platform
```

Comment:
linkage works with `conf.check_cc(lib='dl', mandatory=False)`?

Reply:
Yes that works

Reply:
Could you revert this and add `mandatory=False` flag to libdl check then? I don't think making libdl optional will hurt anything.